gophish-tools/lipca-setup.sh

```diff
@@ -0,0 +1,335 @@
+    case $yn in
+      [Yy]*)
+        sudo vi "$TEMPLATE_PATH/$TEMPLATE_TARGETS_FILENAME"
+        break
+        ;;
+      [Nn]*) echo "Skipping template editing." && break ;;
+      *) echo "Please answer yes or no." ;;
+    esac
+  done
+}
+
+test_by_id_prompt() {
+  # Prompts the user to test campaign by passing an id with passing
+  while true; do
+    read -rp "Do you need to test an existing assessment? (yes/no) " yn
+    case $yn in
+      [Yy]*) read -rp "Enter the ASSESSMENT_ID to test: " id ASSESSMENT_ID="$id" && test_assessment && break ;;
+      [Nn]*) echo "Skipping assessment testing.." && break ;;
+      *) echo "Please specify the ASSESSMENT_ID." ;;
+    esac
+  done
+}
+
+test_post_prompt() {
+  # Prompts the user to ask if the new assessment test should be tested.
+  while true; do
+    read -rp "Do you want to test the assessment? (yes/no) " yn
+    case $yn in
+      [Yy]*)
+        test_assessment
+        break
+        ;;
+      [Nn]*) echo "Skipping assessment tests." && break ;;
+      *) echo "Please answer yes or no." ;;
+    esac
+  done
+}
+
+#=============================
+#     COMPLETE CAMPAIGN
+#=============================
+complete_campaign() {
+  # Run locally pca-gophish-composition/gophish-tools
+  $COMPLETE_CAMPAIGN_PATH "$CAMPAIGN_ID"
+}
+
+complete_campaign_prompt() {
+  # Prompts the user to ask to complete campaign for cleanup.
+  while true; do
+    read -rp "Do you need to complete a previous campaign? (yes/no) " yn
+    case $yn in
+      [Yy]*) read -rp "Enter Campaign ID: " CAMPAIGN_ID CAMPAIGN_ID="$CAMPAIGN_ID" && complete_campaign && break ;;
+      [Nn]*) echo "Skipping campaign completion and proceeding with setup. " && break ;;
+      *) echo "Please answer yes or no." ;;
+    esac
+  done
+}
+
+create_or_manage_prompt() {
+  # Prompts the user for setup or manage actions
+  while true; do
+    read -rp "Do you want to manage an existing campaign or assessment? (yes/no)" yn
+    case $yn in
+      [Yy]*)
+        # Campaign Cleanup Prompt
+        complete_campaign_prompt
+        test_assessment
+        test_prompt
+        break
+        ;;
+      [Nn]*) echo "Proceeding to setup tasks." && break ;;
+      *) echo "Please answer yes or no." ;;
+    esac
+  done
+}
+
+#===========================
+#       Entrypoint
+#===========================
+
+{
+  output_dir_setup
+  logging_setup
+
+  echo "Beginning Li_PCA Setup Process."
+
+  # Export/Complete Prompt
+  # TODO: Determine if we want to force complete campaigns and uncomment.
+
+  # Export_by_id_prompt && complete_campaign_prompt
+  export_by_id_prompt
+
+  # Template Prompt
+  target_template_prompt && email_template_prompt
+
+  # Campaign Setup and Import
+  create_assessment && import_assessment
+
+  # Testing Prompt
+  # TODO: Determine if we want to prompt to test campaign and uncomment.
+  # test_post_prompt
+
+  echo "Li_PCA Setup Process Complete!!"
```

Comment:
Prefer `Li-PCA` to `Li_PCA`:
```suggestion
  echo "Li-PCA Setup Process Complete!!"
```